microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,601 @@
+## Step 3 - Install Mixed Reality Feature Tool and Mixed Reality Toolkit (MRTK)
+
+1. First, [Download the Mixed Reality Feature Tool](https://www.microsoft.com/download/details.aspx?id=102778) from the Microsoft Download Center.
+
+   The Mixed Reality Feature Tool simplifies the process of discovering, installing, and managing Mixed Reality features and packages for your Unity project. For more guidance on installing the Mixed Reality Feature Tool, see [Welcome to the Mixed Reality Feature Tool](/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool).
+
+1. Run the downloaded Mixed Reality Feature Tool executable, and follow the prompts to install the Mixed Reality Toolkit (MRTK) and any additional required packages for HoloLens 2 development.
+
+   The Mixed Reality Feature Tool allows you to select and install the appropriate versions of the Mixed Reality Toolkit (MRTK), along with other Mixed Reality features, directly into your Unity project.
+
+Continue to the next step.
+
+
+<!-- ====================================================================== -->
+## Step 4 - Install the WebView plugin for Unity by using the Mixed Reality Feature Tool
+
+In this step, you'll use the Mixed Reality Feature Tool to install the Microsoft Mixed Reality WebView plugin for Unity.  This will install the specific version of the WebView2 NuGet package that's compatible with HoloLens 2.
+
+1. Launch the Mixed Reality Feature Tool (which you installed above). The **Start** page of the Mixed Reality Feature Tool for Unity opens, containing options to configure settings and to view documentation:
+
+   ![The Configure Settings button in the Mixed Reality Feature Tool](./hololens2-images/mrtk_feature_tool_start_page.png)
+
+1. Click the gear button.  The **Settings** pane opens.
+
+1. Click the **Feature** tab, and then make sure the **Show preview releases** checkbox is selected:
+
+   ![the Feature tab in Settings for the Mixed Reality Feature Tool](./hololens2-images/feature_enable_preview_releases.png)
+
+1. Click the **Ok** button to return to the starting page.
+
+1. Click the **Start** button to begin discovering feature packages.
+
+1. Point the Mixed Reality Feature Tool to your Unity project.  To do this, click the **Browse for the project folder** (**...**) button to the right of the **Project Path** field:
+
+   ![Select the Unity project in the Mixed Reality Feature Tool](./hololens2-images/feature_tool_select_unity_project.png)
+
+1. In the file selection dialog, select your Unity project folder, and then click the **Open** button.
+
+1. In the **Select project** dialog, click the **Discover Features** button.
+
+1. In the **Discover Features** page, expand the **Other features** section, and then select the **Microsoft Mixed Reality WebView** checkbox:
+
+   ![The Discover Features pane in the Mixed Reality Feature Tool](./hololens2-images/mrtk_webview2_selected.png)
```

Comment:
@michaelfarnsworth - add checkbox in capture.  i'll look for one in other pngs

gh rendered section
https://github.com/michaelfarnsworth/edge-developer/blob/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2.md#step-4---install-the-webview-plugin-for-unity-by-using-the-mixed-reality-feature-tool
Find "In the Discover Features page, expand the Other features section, and then select the Microsoft Mixed Reality WebView checkbox:"

png link
https://github.com/michaelfarnsworth/edge-developer/raw/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2-images/mrtk_webview2_selected.png

Reply:
if checkmark looks ok
png link
https://github.com/michaelfarnsworth/edge-developer/raw/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2-images/mrtk_webview2_selected.png
you can close this thread

Reply:
Looks perfect!